_specs/ecip-1001.md

```diff
@@ -0,0 +1,86 @@
+---
+lang: en
+ecip: 1001
+title: ECIP Code of Conduct
+status: Draft
+type: Meta
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/238
+author: Yaz Khoury (@YazzyYaz)
+created: 2019-12-03
+license: CC-By-4.0 License
+---
+This Code of Conduct ECIP has been taken directly from [Contributor Covenant Open Source Code of Conduct](https://www.contributor-covenant.org/)
+under the [CC-By-4.0 License](https://github.com/ContributorCovenant/contributor_covenant/blob/release/LICENSE.md)
+
+This Code of Conduct ECIP applies to all members wishing to participate in the ECIP process:
+1) ECIP Editors
+2) ECIP Coordinators
+3) Hardfork Coordinators
+4) ECIP Authors
+5) Ethereum Classic Community Members Participating in the ECIP Process.
+
+
+## Our Pledge
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards
+Examples of behavior that contributes to a positive environment for our community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
```

Comment:
![Screenshot 2019-12-03 at 10 25 23 PM](https://user-images.githubusercontent.com/6277322/70091179-dfa0b200-161b-11ea-9f14-066e07ba8c7b.png)

Hopefully this can stop personal attacks like seen from the screenshot, which is clearly a violation of this.

Reply:
Out of scope as this Code of Conduct relates to the repository and not Discord.